themes/socialblue/package-lock.json

```diff
@@ -0,0 +1,9873 @@
+{
```

Comment:
Please remember that we use `yarn` and not `npm` for our theme package management.

Reply:
The `package-lock.json` was removed

Reply:
It was fixed in PR #1680